File: jayat_issue_9.c
#include <stdio.h>

int main() {
    int a, b, temp;

    printf("Enter first number: ");
    scanf("%d", &a);
    printf("Enter second number: ");
    scanf("%d", &b);

    printf("\nBefore swapping: a = %d, b = %d\n", a, b);

    temp = a;
    a = b;
    b = temp;

    printf("After swapping using third variable: a = %d, b = %d\n", a, b);

    a = a + b;
    b = a - b;
    a = a - b;

    printf("After swapping without using third variable: a = %d, b = %d\n", a, b);

    return 0;
}

File: jayat_issue_9.py
# Program to swap two numbers using a third variable and without using a third variable

a = int(input("Enter first number: "))
b = int(input("Enter second number: "))

print(f"\nBefore swapping: a = {a}, b = {b}")

temp = a
a = b
b = temp
print(f"After swapping using third variable: a = {a}, b = {b}")

#Reset to original values
a,b=b,a

a = a + b
b = a - b
a = a - b
print(f"After swapping without using third variable: a = {a}, b = {b}")
